frontend/etcdbrowser.js

```diff
@@ -226,4 +226,10 @@ app.controller('NodeCtrl', ['$scope','$http','$cookies', function($scope,$http,$
 
 app.run(function(editableOptions) {
   editableOptions.theme = 'bs3';
-});
+}).directive('propertyForm', function () {
+  return {
+    link: function () {
+      $(".property-form").affix({offset: {top: $(".top-buffer").position().top} }).width($('.tree').width());
+    }
+  }
+});
```

Comment:
I very much like this feature. I have a question about this implementation though (forgive me, I don't know the source code of this project that well): why is adding the CSS not enough? What purpose does this directive serve?

It looks like whenever you click on a leaf node to show the properties, this will append the value of the"top" attribute of the ".top-buffer" to the "top" attribute of the property form and it will set the width of the property form to the width of the tree?

Does this mean the property window will not resize when you resize the window, but the next time you click a leaf node?

Reply:
@mjholtkamp I am also not so good in frontend dev. But as I know it is impossible to do it with CSS only. Because this block stays on some position and starts moving only when scroll come to its top position. With CSS you can make either fixed or flexible block. But not hybrid. 

We need directive because this block doesn't exist in DOM before click. After clicking on leaf old block will be destroyed and new will be created.

I have set the width of fixed block the same with left block because block with property "position: fixed" has different behaviour. And without providing of width size of this block is equal to size of content inside.

Reply:
Discussed offline and verified locally.